### PR TITLE
adding config ability to "Default Location" folder

### PR DESCRIPTION
**Pull request by tcarlsen**
I wanted a way to keep the default folder structure, but at the same time change the "Default Location" name.

Im no python developer, so please let me know if this is the wrong way to handle this or if Im missing something.

### CODE REVIEW

**Comment by Jogai** [comment]
Can we set it to nothing and get no subfolder?

I want to get this (the majority of my photos have no geographical data):

Photos/Sorted/
├── 2003-12-Dec
├── 2004-01-Jan
├── 2005-01-Jan
├── 2005-03-Mar
├── 2005-12-Dec
├── 2009-03-Mar
├── 2009-04-Apr
├── 2018-12-Dec
│   └── Zeeland, The Netherlands
└── 2019-03-Mar
    └── Zeeland, The Netherlands

**Comment by tcarlsen** [comment]
> Can we set it to nothing and get no subfolder?

Maybe, but not quite sure there to put that test

File: Readme.md [hunk 1]
```diff
@@ -237,6 +237,15 @@ full_path=%month/%year/%album|%location|%"Beats me"
 
 What this asks me to do is to name the last folder the same as the album I find in EXIF. If I don't find an album in EXIF then I should use the location. If there's no GPS in the EXIf then I should name the last folder `Beats me`.
 
+#### Custom Default Location
+
+If I wasn't able to find a location for your photo I will use the default location "Unknown Location". This can be changed via your `config.ini` file
+
+```
+[UnknownLocation]
+folder_name=Ukendt placering
```
Comment (jmathai) [inline Readme.md]: Can we reuse `[Directory]` and name the setting `default_folder_name`?

Comment (tcarlsen) [reply Readme.md]: but if I use the `[Directory]` wont that trigger the custom path section of the code? I want to be able to use the default path and only changing the fallback directory

**Comment by tcarlsen** [comment]
seeing how test failed me in the other PR Im not really up for writing one on this one :-(

**Pull request closed without merging**